fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/plan/explain/ExplainPlanVisitorTest.java

```diff
@@ -131,6 +133,13 @@ private static String randomAlphabetic(@Nonnull Random r, int minCount, int maxC
         return new String(letters);
     }
 
+    static Stream<Long> manyRandomSeeds(long seedSeed, int count) {
```

Comment:
This is the one change outside the build system. One of these tests failed during one of the PRB runs, though I've been unable to reproduce. This makes two changes:

1. It updates each test to include its seed in the name so that we can re-run the tests with the same seed if we need to
2. It fixes the seeds when we're not in nightlies so that we get a fixed set of test cases